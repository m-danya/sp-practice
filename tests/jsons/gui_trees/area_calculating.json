{
  "activity_name": "click area test",
  "activity": {
    "root": {
      "class": "class1",
      "clickable": false,
      "children": [
        {
          "class": "class123",
          "clickable": true,
          "children": [],
          "bounds": [0, 0, 200, 200]
        }
      ]
    }
  }
}