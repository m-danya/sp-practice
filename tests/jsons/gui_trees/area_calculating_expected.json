{
    "count": 2,
    "clickable_elements": 1,
    "activity_name": "click area test",
    "element_names": {
        "class1": 1,
        "class123": 1
    },
    "clickable_areas": [40000]
}